fragments/k8s/cdk/README.md

```diff
@@ -1,6 +1,6 @@
 # The Canonical Distribution of Kubernetes
 
-![](https://img.shields.io/badge/kubernetes-1.7.0-brightgreen.svg) ![](https://img.shields.io/badge/juju-2.0+-brightgreen.svg)
+![](https://img.shields.io/badge/kubernetes-1.8.0-brightgreen.svg) ![](https://img.shields.io/badge/juju-2.0+-brightgreen.svg)
```

Comment:
Lets go with just 1.8 instead of a specific patch level so we don't have to increment /every/ time